src/DynamoCoreWpf/Views/PackageManager/Pages/PublishPackagePreviewPage.xaml.cs

```diff
@@ -1,15 +1,38 @@
+using System.ComponentModel;
 using System.Windows;
 using System.Windows.Controls;
 
+
 namespace Dynamo.PackageManager.UI
 {
     /// <summary>
     /// Interaction logic for PublishPackagePreviewPage.xaml
     /// </summary>
-    public partial class PublishPackagePreviewPage : Page
+    public partial class PublishPackagePreviewPage : Page, INotifyPropertyChanged
```

Comment:
Would it be counted as API break? I am extending additional interface, but that wouldn't reflect in changes to the way the class is instantiated.. right? Otherwise, all tests pass here, if this doesn't count as API break, then we can proceed?

Reply:
Looked up in https://stackoverflow.com/questions/1456785/a-definitive-guide-to-api-breaking-changes-in-net, this case seems to be allowed